src/pages/library/Code.js

```diff
@@ -61,7 +61,10 @@ export const DockstoreTile = () => {
   return div({ style: { display: 'flex' } }, [
     h(LogoTile, { logoFile: dockstoreLogo, style: { backgroundColor: 'white' } }),
     div([
-      h(Link, { href: `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=WDL&searchMode=files` }, 'Dockstore'),
+      h(Link, {
+        href: `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=WDL&searchMode=files`,
+        style: { color: colors.accent(1.1) } // For a11y, we need at least 4.5:1 contrast against the gray background
```

Comment:
I discussed this with @s-rubenstein in a different PR. In this particular case, this link is on top of a gray background rather than a white one, so the link color has to be darkened a bit to maintain the contrast ratio.

But I agree it would make sense to re-evaluate the site colors in the context of contrast ratios. If the colors were darker to begin with, they might work on a gray background without having to darken them further.